activesupport/lib/active_support/deprecation/reporting.rb

```diff
@@ -36,6 +36,15 @@ def silence
         @silenced = old_silenced
       end
 
+
+      # Outputs a deprecation warning with 'will be removed' message to
```

Comment:
@davydovanton what do you think about giving a more realistic example like @matthewd suggests?

@matthewd I have no idea how popular the symbol option is. If we're giving a more realistic example, it would probably make more sense to add a more descriptive message too, yeah.